### DISCUSSION TITLE
Readwise Reader

### DISCUSSION THREAD

**Discussion by goooseman** [catppuccin/catppuccin#2478] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

The App.css file [example](https://read.readwise.io/assets/App.c5c6ff72.css) have a lot of different variables that may be themed according to Catppuccin.

Also some unfinished example of this app theming may is found on userstyles: https://userstyles.world/style/12382/half-baked-readwise-reader-sepia-theme.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Is this a duplicate of #2479 ?
